app/lib/frontend/templates/landing.dart

```diff
@@ -3,26 +3,60 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import '../../package/models.dart' show PackageView;
+import '../../shared/tags.dart';
 import '../../shared/urls.dart' as urls;
 
+import '../request_context.dart';
+
 import '_cache.dart';
 import 'layout.dart';
 import 'misc.dart' show renderMiniList;
 
 /// Renders the `views/page/landing.mustache` template.
 String renderLandingPage({
-  List<PackageView> taggedPackages,
-  List<PackageView> topPackages,
+  List<PackageView> ffPackages, // old + new
+  List<PackageView> topPackages, // old only
+  List<PackageView> mostPopularPackages, // new only
+  List<PackageView> topFlutterPackages, // new only
+  List<PackageView> topDartPackages, // new only
 }) {
-  final hasTagged = taggedPackages != null && taggedPackages.isNotEmpty;
-  final hasTop = topPackages != null && topPackages.isNotEmpty;
+  final isExperimental = requestContext.isExperimental;
+  final hasTagged =
+      !isExperimental && ffPackages != null && ffPackages.isNotEmpty;
+  final hasTop =
```

Comment:
Maybe we should make a local utility function:

`bool nullOrEmpty(List l) => !isExperimental && l == null && l.isNotEmpty`